easybuild/main.py

```diff
@@ -219,9 +221,68 @@ def main(args=None, logfile=None, do_build=None, testing=False):
     eb_cmd_line = eb_go.generate_cmd_line() + eb_go.args
     log_start(eb_cmd_line, eb_tmpdir)
 
+    # report on umask here so we don't have to forward this info to `main`
     if options.umask is not None:
         _log.info("umask set to '%s' (used to be '%s')" % (oct(new_umask), oct(old_umask)))
 
+    # sanity check
+    if options.daemonize and options.logtostdout:
```

Comment:
Since all of the options are attributes in `self.options`, I think you need to resort to something like:

```python
list_avail_option_attrs = [a for a in dir(self.options) if a.startswith('list_') or a.startswith('avail_')]
if any([getattr(self.options, a) for a in list_avail_option_attrs]):
    ....
```